content-repo/post_github_comment.py

```diff
@@ -96,22 +96,34 @@ def post_comment(netlify_deploy_file: str):
     token = os.getenv('GITHUB_TOKEN')
     if not token:
         raise ValueError("Can't post comment. GITHUB_TOKEN env variable is not set")
-    with open(netlify_deploy_file, 'r') as f:
-        netlify_info = json.load(f)
-    deplpy_url = netlify_info['deploy_url']
+
+    # handle Netlify deployment message
+    if netlify_deploy_file.endswith(".json"):
+        with open(netlify_deploy_file, 'r') as f:
+            netlify_info = json.load(f)
+        deploy_url = netlify_info['deploy_url']
+
+    # handle Firebase deployment message
+    else:
+        with open(netlify_deploy_file, 'r') as f:
+            if matched_url := re.search("https://xsoar-pan-dev--pull-request-.*web.app", f.read()):
+                deploy_url = matched_url.group(0)
+            else:
+                raise ValueError("Can't post comment. Deployment was unsuccessful: \n" + f.read())
+
     # preview message
     message = "# Preview Site Available\n\n" \
```

Comment:
add to the title the site hosting (netlify/firebase) at least until we finish the migration and remove the double comments